#!/usr/bin/env python
PACKAGE = "teamplay"

import roslib;
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("shootPathWidth", double_t, 0, "shoot path width in meters", 0.0, 0.0, 10.0)
gen.add("straightShotThreshold", double_t, 0, "straight shot threshold in meters", 0.0, 0.0, 11.0)
gen.add("aimForCornerThreshold", double_t, 0, "aim for corner threshold in meters", 0.0, 0.0, 11.0)
gen.add("shotThreshold", double_t, 0, "shot threshold in meters", 0.0, 0.0, 25.0)

exit(gen.generate(PACKAGE, "teamplayNode", "teamplayShooting"))
